sbin/repair_services.sh

```diff
@@ -1,36 +1,122 @@
 #!/usr/bin/env bash
 # shellcheck enable=all
 
-#### This script repairs core services that are not currently running
+#### This script repairs core services that are not currently active
 
-repair_services() {
-    services=(
-        "nginx"
-        "apache2"
-        "postgresql"
-        "nullsmtpd"
-        "submitty_websocket_server"
-        "submitty_autograding_worker"
-        "submitty_autograding_shipper"
-        "submitty_daemon_jobs_handler"
-    )
-
-    for service in "${services[@]}"; do
-        if ! sudo systemctl is-active --quiet "${service}"; then
-            today=$(date +%Y%m%d)
-            log_file="/var/log/services/${today}.txt"
-            last_status=$(sudo systemctl status "${service}")
-
-            if [[ ! -f "${log_file}" ]]; then
-                sudo touch "${log_file}"
+declare -A services=(
+    ["nginx"]="Nginx Server"
+    ["apache2"]="Web Server"
+    ["postgresql"]="Database Server"
+    ["nullsmtpd"]="Mail Server"
```

Comment:
This is the FAKE mail server.  It only runs on development machines.  This service does not run on a production server.  So I don't think it should be included in this list.  Or perhaps it should have an 'if vagrant' wrapped around it.  At least the comment hsould make clear its not the real mail server.

Reply:
And I see that we have lots of ineffective attempts to start this service on the production machine...
oops.